code/script.js

```diff
@@ -1,19 +1,161 @@
-// Start here
+// Start here - with if/else and arrays 
+
 
 // Step 1 - Welcome and introduction
 // Your code goes here
 alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
 )
+let person = prompt(`Please enter you name`);
+
+if (person != null) {
+  alert(`Hello ${person}! that is a really cool name!`);
+}
 
```

Comment:
Hi Isa! Hope everything is well :) I like that its really easy to understand your code from the start!